cfme/services/myservice/ssui.py

```diff
@@ -75,20 +75,20 @@ class EditMyServiceView(ServiceEditForm):
     @property
     def is_displayed(self):
         return (
-            self.in_myservices and
+            self.name.is_displayed and
```

Comment:
Question: Why in_myservices was replaced?

Reply:
Edit service opens a new window on top of the page so in_myservice is False .